frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.module.css

```diff
@@ -0,0 +1,11 @@
+:root {
+  --mb-color-embedding-option-primary: var(--mb-color-icon-primary);
+  --mb-color-embedding-option-secondary: var(--mb-color-icon-secondary);
+}
```

Comment:
nit: I wouldn't pollute the global scope. Since these colors only exist on the embedding option context. I'd rather have this under a class. What do you think?

Reply:
YES!